include/internal/yamerl_constr.hrl

```diff
@@ -43,14 +43,14 @@
 %% -------------------------------------------------------------------
 
 -record(unfinished_node, {
-    module = ?MODULE,
-    path,
-    pres,
-    priv
+    module = ?MODULE :: atom(),
+    path  :: tuple(),
+    pres  :: [tuple()],
+    priv  :: tuple() | map() | list()
```

Comment:
Perhaps this should be `any()` instead of `tuple() | map() | list()`, what do you think?

Reply:
Thank you, agree, I have committed this change, please review again.

Btw, if we can release a new version that would be appreciated.